src/Microsoft.VisualStudio.Services.Agent/Util/IOUtil.cs

```diff
@@ -124,11 +120,10 @@ public static void DeleteDirectory(string path, CancellationToken cancellationTo
             // could get out of control for a large directory with access denied on every file.
             using (var tokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken))
             {
-                // Delete all files and store all subdirectories.
+                // Delete all files and store all subdirectories.\
                 directory
                     .EnumerateFileSystemInfos("*", SearchOption.AllDirectories)
                     .AsParallel()
```

Comment:
Not sure why this \ is present. I'm also not sure I follow your comment about the cancellation token. Would you elaborate?